mono/metadata/object.c

```diff
@@ -4395,76 +4395,43 @@ mono_runtime_try_run_main (MonoMethod *method, int argc, char* argv[],
 	return mono_runtime_try_exec_main (method, args, exc);
 }
 
-
 static MonoObject*
-serialize_object (MonoObject *obj, gboolean *failure, MonoObject **exc)
+serialize_or_deserialize_object (MonoObject *obj, MonoError *error,
+	const gchar *method_name, MonoMethod **method)
 {
```

Comment:
error is usually passed as the last argument.

Reply:
I was kinda appending parameters, albeit having replaced failure/exc with error; ok.